content/en/blog/_posts/2023-04-11-volume-group-snapshot-alpha.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: blog
+title: "Kubernetes 1.27: Volume Group Snapshot Moves to Alpha"
```

Comment:
Do we have some of this details on the k8s.io/docs website?

Reply:
The doc will be at the CSI driver doc website: https://kubernetes-csi.github.io/docs/

Reply:
We don't have it yet. We'll have the doc after external-snapshotter and other CSI sidecars are released which will happen after K8s 1.27 release.

Reply:
Added a WIP doc PR here: https://github.com/kubernetes-csi/docs/pull/557